src_assets/common/assets/web/configs/tabs/Advanced.vue

```diff
@@ -0,0 +1,104 @@
+<script setup>
+import { ref } from 'vue'
+import PlatformLayout from '../../PlatformLayout.vue'
+
+const props = defineProps([
+  'platform',
+  'config',
+  'global_prep_cmd'
+])
+
+const config = ref(props.config)
+
+</script>
```

Comment:
```suggestion
<script setup>
  import { ref } from 'vue'
  import PlatformLayout from '../../PlatformLayout.vue'

  const props = defineProps([
    'platform',
    'config',
    'global_prep_cmd'
  ])

  const config = ref(props.config)
</script>
```

Could you make this change across all the vue files?

Reply:
@ReenigneArcher For some reason that's the default [vue style](https://vuejs.org/guide/introduction.html#composition-api), using no spaces on the script block, but I see no reason not to change

Reply:
We'll be using the vue docs style